src/dg/dg_base.cpp

```diff
@@ -1282,12 +1277,11 @@ void DGBase<dim,real,MeshType>::assemble_residual (const bool compute_dRdW, cons
             timer.stop();
             assemble_residual_time += timer.cpu_time();
         }
-    } catch(...) {
-        assembly_error = 1;
-    }
+    // } catch(...) {
+    //     assembly_error = 1;
+    // }
```

Comment:
without this turned off, its impossible to debug the code -- let me know how you think we should handle this

Reply:
From discussion a few weeks ago, there seemed to be a way to get around this. Could you test that and (if successful), revert this?

Reply:
@dougshidong what was the command you mentioned?